packages/db/queries/get_env_with_build.sql

```diff
@@ -0,0 +1,8 @@
+-- name: GetEnvWithBuild :one
+SELECT sqlc.embed(e), sqlc.embed(eb)
+FROM "public"."envs" e
+LEFT JOIN "public"."env_aliases" ea ON ea.env_id = e.id
```

Comment:
if you want aliases, you will have to do it little bit differently. I think you can do subquery and aggregate them to array